src/main/groovy/com/avioconsulting/mule/linter/model/ConfigurationFile.groovy

```diff
@@ -23,6 +24,54 @@ class ConfigurationFile extends ProjectFile {
         }
     }
 
+    Boolean doesExists() {
+        return exists
+    }
+
+    List<MuleComponent> findChildComponents(Map<String, String> elements, Boolean include) {
```

Comment:
Can you add javadocs/groovydocs to describe what these arguments do?  Its not clear from the method and argument names.

Reply:
added.